verification/curator-service/api/src/util/validate-env.ts

```diff
@@ -26,10 +26,12 @@ export default function validateEnv(): Readonly<{
         PORT: port({ default: 3001 }),
         GOOGLE_OAUTH_CLIENT_ID: str({
             desc: 'OAuth client ID from the Google developer console',
+            default: 'replace to enable auth',
```

Comment:
no please outside of dev we have to make sure we put in the right values here, same below

Reply:
Done